src/views/dashboards/PasswordManager/BulkPasswordAccountModal.vue

```diff
@@ -0,0 +1,180 @@
+<template lang="pug">
+.modal.fade(
+  tabindex='-1'
+  role='dialog'
+  aria-labelledby='bulk-password-account-modal'
+  aria-hidden='true'
+  v-loading="globalLoading")
+    .modal-dialog.modal-lg
+      .modal-content
+        .modal-header
+          h3.modal-title.d-flex.align-items-center #[i.now-ui-icons.users_circle-08.mr-2] Bulk Upload
+          button.close(type='button' data-dismiss='modal' aria-label='Close')
+            span(aria-hidden='true') &times;
+        .modal-body
+          div.text-center
+            p Upload a .csv of at least five accounts to add to Password Manager. #[b Format should follow table below:]
+          form(@submit.prevent="sendAccountsToBlockchain")
+            div
+              div.text-center
+                input.form-control.input-block.mr-2(
+                  :id="`bulk-upload-file-${uid}`"
+                  type="file",
+                  @change="parseFile")
+                button.btn.btn-primary(
+                  v-loading="globalLoading"
+                  :disabled="globalLoading"
+                  @click.prevent="triggerFile")
+                    | #[i.now-ui-icons.arrows-1_share-66] Upload CSV
+
+              div.mt-2.table-responsive
+                table.table.table-striped.table-bordered.m-0
+                  thead
+                    tr
+                      th Account Name
+                      th Username
+                      th Password
+                      th Additional Info
```

Comment:
Can we add a "remove" column with a red X to remove the item if a user wants or something is not correct by mistake?

Reply:
5790096